labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -50,7 +83,7 @@ object CollectionExercise02 {
    * using a functional approach.
    */
   def groupAdultsPerAgeGroup(persons: Seq[Person]): Map[Int, Seq[Person]] = {
-    error("fix me")
+    persons filter ((A) => A.age >= 18) sortBy(_.name) groupBy(_.age/10*10) 
```

Comment:
I like how you played around with syntax freedom here, because I did the same throughout the assignment. I kept switching from using the dot operator to just using spaces.